mod/messages/start.php

```diff
@@ -415,6 +415,130 @@ function messages_get_unread($user_guid = 0, $limit = null, $offset = 0, $count
 	return elgg_get_entities_from_metadata($options);
 }
 
+
+/**
+ * Returns the unread messages from other users in a user's inbox
```

Comment:
Just for us it's not really necessary but if someone wants to replicate this menu split without using our skin then it will be needed.